### ISSUE TITLE
[Combobox] Combobox 'Disabled' has information in tags hidden with "aria-hidden"

### ISSUE THREAD

**Issue by a11ymiko** [sl-design-system/components#3008]
## Current behavior
Inside Disabled Combobox there are `<sl-tag>` that have `aria-hidden='true'` and are placed inside `<sl-tag-list aria-label='Selected options'>` which does not have `aria-hidden` attribute. 
Because of that both NVDA and VoiceOver in reading mode (using arrow keys) can be placed on `<sl-tag-list>` element and announce "List, Selected options, 0" which is not true. User can't place screen reader on each tag inside the list. 
<img width="578" height="48" alt="Image" src="https://github.com/user-attachments/assets/9a70b588-f40a-4a08-b760-ecf5d371d463" />

## Proposed solution
`<sl-tag>` inside `<sl-tag-list>` should not have `aria-hidden='true'` and should be announced as list items. 
This recommendation works **only for Disabled Combobox** as in these case tags are used to inform user about what is selected and cannot be changed, which is important information.

### 👤 Your name

Kacper

### 🧢 Your Product/Team

SLDS
